subway_prediction_ml/test/test_actuals_adder.py

```diff
@@ -0,0 +1,42 @@
+import os
+import pandas as pd
+import sys
+import unittest
+
+sys.path.append(os.path.abspath(os.path.join(__file__, '..', '..')))
+from actuals_adder import ActualsAdder
+
+
+class TestActualsAdder(unittest.TestCase):
+    def test_joins_actuals_to_vehicle_datapoints(self):
+        actuals_data = pd.DataFrame(
+            columns=['trip_id', 'stop_id', 'time'],
+            data=[
+                ['B-111', '70000', 1559740000],
+                ['B-111', '70002', 1559740060],
+                ['B-111', '70004', 1559740120],
+                ['B-112', '70100', 1559740000],
+                ['B-112', '70102', 1559740060],
+                ['B-112', '70104', 1559740120]
+            ]
+        )
+
+        vehicle_data = pd.DataFrame(
+            columns=['gtfs_trip_id', 'timestamp'],
+            data=[
+                ['B-111', "2019-06-05T10:24:30.000000Z"],
```

Comment:
Since this step is largely about the n*m join between actuals and vehicles, can you add another test vehicle point for `B-111`, maybe in between when it's at 70000 and 70002, just so we can see the actuals get repeated for both datapoints (and the actual in the past get dropped)?